beeline/src/java/org/apache/hive/beeline/BeeLine.java

```diff
@@ -182,6 +182,7 @@ public class BeeLine implements Closeable {
   private static final String HIVE_CONF_PREFIX = "--hiveconf";
   private static final String PROP_FILE_PREFIX = "--property-file";
   public static final String PASSWD_MASK = "[passwd stripped]";
+  public static final String BEELINE_DEFAULT_OPERATIONAL_LOG_LEVEL = "EXECUTION";
```

Comment:
keep it simple: DEFAULT_OPERATIONAL_LOG_LEVEL, it's inside of BeeLine class

Reply:
Done